spec.md

```diff
@@ -550,6 +552,16 @@ schema being used.
 | version | string | This MUST be a string conforming to a semantic version. The Platform MAY use this field to determine whether a maintenance update is available for a Service Instance. |
 | description | string | This SHOULD be a string describing the impact of the maintenance update, for example, important version changes, configuration changes, default value changes, etc. The Platform MAY present this information to the user before they trigger the maintenance update. |
 
+##### Extensions Object
+
+| Response Field | Type | Description |
+| --- | --- | --- |
+| id* | string | A Uniform Resource Name ([URN](https://tools.ietf.org/html/rfc2141)) that uniquely identifies the extension. It MUST include the namespace identifier(NID) `osbext` and a specific string(NSS) for the extension. For example `urn:osbext:/v1/backup`. |
```

Comment:
shouldn't this be `and a NAMESPACE specific string(NSS)` (emphasis mine)?

Reply:
Yes it could be if we think it will makes it more clear.
It's a reference to the URN syntax and the parts it consist of. https://tools.ietf.org/html/rfc2141#section-2

Reply:
just writing down some concerns from the OSB meeting:
- is this ID intended to be unique PER BROKER, or globally unique among ALL BROKERS when pointing to a specific extension? (i.e. all extensions that point http://example.com/extensions/backup_restore.yaml on ALL brokers are intended to be named the same thing on all brokers?)

Reply:
We have two main reasons for an identifier:

The first is to support the use-case of the community coming up with a standard extensions and perhaps writing platform specific tooling for these extensions. Brokers may then want to opt-in to that already defined extension. 

Then we have a separate requirement that a broker itself needs to be able to namespace extensions that it provides, say a broker wants to provide two backup and restore options and the openapi documents have colliding paths. 

We were hoping that having all extensions provide a globally unique urn would solve both of these problems, so I imagined that the ID (URN) was globally unique. I think perhaps our example here is misleading.

As for the structure of the URN this is something I'm not too sure about. I imagine the domain should be in there e.g. `urn:osbext:pivotal.io/v1/mysqlbackups` or `urn:osbext:openservicebrokerapi.org/v1/community-back`. Open to ideas around what this should look like and if URNs are the right thing here.

Reply:
I also think the recommendation should be something like `urn:osbext:<domain>/<some string>` . But whatever we do, all the examples in the spec should follow the recommended pattern.

Reply:
I've changed all examples to follow the pattern: `urn:osbext:some-domain:some-version`